File: Secretary.API/Models/Grupo.cs
using System;
using System.Collections.Generic;

namespace Secretary.API.Model
{
    public partial class Grupo
    {
        public Grupo()
        {
            Publicador = new HashSet<Publicador>();
        }

        public long Id { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string Ipaddress { get; set; }
        public long AuditoriaUsuario { get; set; }
        public string Local { get; set; }
        public long? AjudanteId { get; set; }
        public long CongregacaoId { get; set; }
        public long? SuperintendenteId { get; set; }

        public Publicador Ajudante { get; set; }
        public Congregacao Congregacao { get; set; }
        public Publicador Superintendente { get; set; }
        public ICollection<Publicador> Publicador { get; set; }
    }
}

File: Secretary.SPA/src/app/_models/Grupo.ts
import { Publicador } from './Publicador';
import { Congregacao } from './Congregacao';

export interface Grupo {
  id: number;
  local: string;
  ajudanteId: number;
  ajudante: Publicador;
  congregacaoId: number;
  congregacao: Congregacao;
  superintendenteId: number;
  superintendente: Publicador;
}
